hack/make-pjs.sh

```
@@ -0,0 +1,191 @@
+function branch_info() {
+  git ls-remote "https://github.com/${ORG}/${REPO}" "$BRANCH"
+}
+
+# Generate PJs manifests and save them in a temporary directory /tmp/tests-XXXXX
+function make_prowjobs() {
+  tests=$1
+
+  base_dir=$(mktemp -p /tmp -d 'tests-XXXXX')
+  mkdir -p "$base_dir"=
+
+  mkpj_img=$(awk -F '=' '/mkpj/{print $2}' ${RELEASE_REPO}/hack/images.sh)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)